docs/aws/aws-how-to/instances/find-ubuntu-images.rst

```diff
@@ -10,73 +10,162 @@ On AWS, cloud images are referred to as Amazon Machine Images (AMIs). Canonical
 All images mentioned below are also available in `AWS Outposts <https://aws.amazon.com/outposts/>`_.
 
 
-Finding images with SSM
------------------------
-The SSM Parameter Store is a hierarchical data service provided by AWS for configuration management. It can be used to store passwords, license codes, configuration strings, Amazon Machine Image (AMI) IDs, and more. Canonical provides a set of publicly available parameters in the parameter store under the hierarchy ``/aws/service/canonical``. One useful set of parameters available under that hierarchy is the set of latest AMI IDs for Ubuntu images. These IDs can be found programmatically using the AWS CLI.
+Finding images for EC2 and EKS
+------------------------------
+.. tabs::
+   .. tab:: Using SSM Parameter Store
```

Comment:
i like the idea, but the actual rendering is a little awkward. when you expand the tab its not immediately obvious there is a sub tab. @k-dimple may have better suggestions, but there are a couple routes i can think of here

* maintain separate tables and allow the sections to be collapsed to easily hide what you don't care about
* have separate pages for each method. this isn't great for existing users, but it makes it much clearer what the use case. however, most users likely don't know about ssm parameters (and this is really what we want to push imo). not a strong advocate here

Reply:
went through this in a 1:1 and it appears to be a desktop settings problem. if i have dark mode enabled on the host system then it renders as follows. fine to proceed here.

<img width="960" height="445" alt="image" src="https://github.com/user-attachments/assets/07a0c088-55a0-4317-968c-5217a2406361" />